### ISSUE TITLE
Switches changing colour?

### ISSUE THREAD

**Issue by kochie** [fannypackui/fannypack#37]
The colours on the switch seem to change to the default theme when clicked.
![fannypack_switch](https://user-images.githubusercontent.com/10809884/50618597-27204900-0f48-11e9-85b3-d952ad79996c.gif)
You can see it in the [docs](https://fannypack.style/form/switch#states)

**Comment by jxom** [comment]
Hey @kochie,

This is expected, as the Switch gains focus when it is clicked, triggering the focus outline. [This is important for accessiblity](https://developers.google.com/web/fundamentals/accessibility/focus/). This is also evident with [input fields](https://fannypack.style/form/input#states). However, maybe we should make this a bit clearer? Adding a focus outline (of the same colour - maybe a lighter variant?) around the state outline could be a better solution... Thoughts?

**Comment by kochie** [comment]
Yeah, that seem reasonable, I would expect that the focus colour for the switch would be the same as the switch colour or lighter as it's a bit jarring.

**Comment by terencehuynh** [comment]
Lighter/darker variant of the same colour would be nice - especially given we have associated the colours with meanings (i.e. red for danger, yellow for warning).
